### PR TITLE
FF and Safari 'use strict'; mode by default; can't nest functions in …

### PR DESCRIPTION
**Pull request by FBRTMaka**
…conditionals

@DanielJMaher , @birdage

### CODE REVIEW

**Comment by DanielJMaher** [comment]
@birdage This make sense?

**Comment by FBRTMaka** [comment]
@DanielJMaher he's out

File: ooiui/static/js/views/common/MapView.js [hunk 1]
```diff
@@ -164,25 +164,23 @@ var MapView = Backbone.View.extend({
...
+            return p;
+        };
         var golden_ratio_conjugate = 0.618033988749895;
         var h = Math.random();
 
         var hslToRgb = function (h, s, l){
             var r, g, b;
 
-            if(s == 0){
+            if(s === 0){
```
Comment (birdage) [inline ooiui/static/js/views/common/MapView.js]: @FBRTMaka whats this extra `=` for?

Comment (FBRTMaka) [inline ooiui/static/js/views/common/MapView.js]: @birdage a triple equals in JS checks for both content AND type. A double equals only checks for content.  It's 'typically' considered good practice to use the triple equals (and, converse equivalent of !== ) when doing conditional checks.

Comment (birdage) [inline ooiui/static/js/views/common/MapView.js]: @FBRTMaka thanks, im familiar with the syntax, i was wondering why we changed it.

Comment (FBRTMaka) [inline ooiui/static/js/views/common/MapView.js]: @birdage Ah, I see.  Yeah, same reason applies ;)

**Pull request merged**